sql_generators/funnels/templates/funnel.sql

```diff
@@ -20,9 +20,9 @@
         {% endfor %}
       {% endif %}
       {{ data_sources[steps[step_name].data_source].submission_date_column }} AS submission_date,
-      {{ data_sources[steps[step_name].data_source].client_id_column }} AS client_id,
+      {{ data_sources[steps[step_name].data_source].client_id_column }} AS client_id_column,
```

Comment:
This and similar change below are needed because `client_id` is a top-level column in `events_stream` tables and becomes ambiguous here.